packages/integration-karma/test/template/directive-if/index.spec.js

```diff
@@ -126,6 +127,28 @@ describe('if:true directive', () => {
                 });
         });
     }
+
+    it('should render slot content multiple levels deep when toggled 2 times', () => {
+        const elm = createElement('x-multiple-slot', { is: MultipleSlot });
+        document.body.appendChild(elm);
+        elm.shadowRoot.querySelector('.textToggle').click();
+
+        return Promise.resolve()
+            .then(() => {
+                expect(elm.shadowRoot.textContent).toContain('text in multiple level slot');
+                // hide the slot
+                elm.shadowRoot.querySelector('.textToggle').click();
+                return Promise.resolve();
+            })
+            .then(() => {
+                // show the slot
```

Comment:
Assert the `textContent` is now `''`.

Reply:
wow, this one was tricky in native shadow thanks! 👍